IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormFileHistoryTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using CommonTestUtils;
+using CommonTestUtils.MEF;
+using FluentAssertions;
+using GitCommands;
+using GitUI;
+using GitUI.CommandsDialogs;
+using GitUIPluginInterfaces;
+using Microsoft.VisualStudio.Composition;
+using NUnit.Framework;
+
+namespace GitExtensions.UITests.CommandsDialogs;
+
+[Apartment(ApartmentState.STA)]
+public class FormFileHistoryTests
+{
+    [SetUp]
+    public void SetUp()
+    {
+        AppSettings.UseBrowseForFileHistory.Value = false;
+
+        TestComposition? composition = TestComposition.Empty
+            .AddParts(typeof(MockLinkFactory));
```

Comment:
I think this needs to be this to keep tests happy:
```suggestion
            _composition = TestComposition.Empty
                .AddParts(typeof(MockLinkFactory))
                .AddParts(typeof(MockWindowsJumpListManager))
                .AddParts(typeof(MockRepositoryDescriptionProvider))
                .AddParts(typeof(MockAppTitleGenerator));
```

Reply:
Yeah, that's a part of the test class which I don't understand at all - I've added a minimum code for the tests to run (and satisfy the _you ain't gonna need it_ principle) but perhaps it is _you **are** gonna need it_ after all.